pkg/apiserver/api_installer.go

```diff
@@ -567,6 +571,24 @@ func (a *APIInstaller) registerResourceHandlers(path string, storage rest.Storag
 			}
 			addParams(route, action.Params)
 			ws.Route(route)
+		case "DELETECOLLECTION":
+			doc := "delete collection of " + kind
+			if hasSubresource {
+				doc = "delete collection of " + subresource + " of a " + kind
```

Comment:
ok, just wanted to ensure i wasn't crazy that this was not yet actually supported, but i can see how a kind may manage a collection of subresources and we would want delete on that collection.